processes/burntmapping_chunks.json

```diff
@@ -0,0 +1,196 @@
+{
+  "namespace": "user",
+  "description": "Classical Normalized Burnt Ratio(NBR) difference is performed using VITO backend on a chunk polygon.",
```

Comment:
I think community examples should not be back-end specific.

Reply:
corrected